### PR TITLE
LPS-171635 [PRM] Change the getMaxDateActivity.ts and getMinDateActivity.js logic

### PR DESCRIPTION
**Pull request by kiwm**
Related issue: https://issues.liferay.com/browse/LPS-171635

### CODE REVIEW

**Comment by kiwm** [comment]
ci:test:sf

**Comment by kiwm** [comment]
ci:test:relevant

**Comment by kiwm** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/getMaxDateActivity.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
  *
  * The contents of this file are subject to the terms of the Liferay Enterprise
  * Subscription License ("License"). You may not use this file except in
- * compliance with the License. You can obtain a copy of the License by
+ * compliance with the License. You can obtainP a copy of the License by
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/utils/getMaxDateActivity.ts:6]:
```suggestion
 * compliance with the License. You can obtain a copy of the License by
```

**Review by thaynam — CHANGES_REQUESTED** [review]

**Comment by thaynam** [comment]
ci:test:sf

**Review by thaynam — APPROVED** [review]

**Comment by thaynam** [comment]
ci:forward

**Comment by kiwm** [comment]
ci:test:relevant

**Comment by thaynam** [comment]
ci:forward

**Pull request closed without merging**